lib/stream.js

```diff
@@ -126,7 +126,7 @@ module.exports = class UDXStream extends streamx.Duplex {
     this.emit('connect')
   }
 
-  async send (buffer) {
+  send (buffer) {
```

Comment:
If you remove async you need to change the return false to be be return Promise.resolve(false)

Reply:
Actually just keep it async as we need to wrap errors also otherwise

Reply:
I thought it was a bit crazy to remove the async but I wasn't sure why, now I know